<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MainMenu" name="mainMenuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>301</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="RoomCreationWidget" name="roomCreationWidget" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>39</y>
      <width>491</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="JoiningWidget" name="joiningWidget" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>410</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="GameWidget" name="gameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="LoginWidget" name="loginWidget" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="SplashScreenWidget" name="splashScreenWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>240</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="RoomsListWidget" name="roomsListWidget" native="true">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>210</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="GameConfirmationWidget" name="gameConfirmationWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>440</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainMenu</class>
   <extends>QWidget</extends>
   <header>mainmenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RoomCreationWidget</class>
   <extends>QWidget</extends>
   <header>roomcreationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JoiningWidget</class>
   <extends>QWidget</extends>
   <header>joiningwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameWidget</class>
   <extends>QWidget</extends>
   <header>gamewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoginWidget</class>
   <extends>QWidget</extends>
   <header>loginwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SplashScreenWidget</class>
   <extends>QWidget</extends>
   <header>splashscreenwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RoomsListWidget</class>
   <extends>QWidget</extends>
   <header>RoomsListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameConfirmationWidget</class>
   <extends>QWidget</extends>
   <header location="global">GameConfirmationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
